### PR TITLE
dns-forward: add ppx_cstruct dep

### PR DESCRIPTION
**Pull request by avsm**
cc @djs55

### CODE REVIEW

**Comment by avsm** [comment]
hm travis build failure on mirage_flow for some reason -- something to do with new mirage-flow release @samoht?

File: packages/dns-forward/dns-forward.0.7.2/opam [hunk 1]
```diff
@@ -18,6 +18,7 @@ depends: [
   "ocamlbuild" {build}
   "topkg"      {build}
   "ppx_tools"  {build}
+  "ppx_cstruct" {build}
   "cmdliner"
   "mirage-flow" {<= "1.1.0"}
```
Comment (samoht) [inline packages/dns-forward/dns-forward.0.7.2/opam]: This should be `{= "1.1.0"}`

**Comment by samoht** [comment]
It's not related to the recent release, it's a missing lower bound.

**Comment by avsm** [comment]
thanks, pushed a fix. cc @djs55 to see if there an upstream fix to relax the specific mirage-flow bound

**Comment by samoht** [comment]
Merging, and I let @djs55 report the constraints upstream (and maybe make a new release without them :-) )

**Comment by djs55** [comment]
I think I need to update `vpnkit` to Mirage 3 before I can update `dis-forward` :/ At least no-one else uses `dns-forward` I think.

**Pull request merged**